assets/css/structure.css

```diff
@@ -0,0 +1,69 @@
+body {
+  font-family: var(--primary-font);
+  background-color: var(--primary-color);
+  place-items: center;
+}
+
+.site-wrapper {
+  max-width: 1200px;
+  padding: 0 var(--spacing-small);
+  margin: 0 auto;
+}
+
+.content {
+  height: 100vh;
+  background-color: var(--sixth-color);
+  display: grid;
+  place-items: center;
+  border: var(--primary-border);
+  border-radius: var(--sizing-small);
+}
+
+.title {
+  font-size: var(--bolder-font);
+  font-weight: bolder;
+  color: var(--primary-color);
+}
+
+.filter-list {
+  display: grid;
+  grid-template-columns: auto auto auto;
+}
+
+.filter-item {
+  color: var(--primary-color);
+  padding: var(--spacing-xxlarge);
+}
+
+.todo-item-add {
+  width: 350px;
+  border: var(--secondary-border);
+  border-radius: var(--sizing-xsmall);
+  padding: var(--spacing-small);
+  margin: var(--spacing-small);
+  display: flex;
+  justify-content: space-between;
+}
+
+.todo-item {
+  width: 350px;
+  border: var(--secondary-border);
+  border-radius: var(--sizing-xsmall);
+  padding: var(--spacing-small);
+  margin: var(--spacing-small);
+}
+
+input {
+  accent-color:var(--secondary-color);
+  border: none;
+}
+
+.fa-solid {
+  color: var(--quaternary-color);
+  font-size: var(--bold-font);
+  visibility: hidden;
```

Comment:
please remove this. add icon always should be visible. for edit and delete we should do that.

Reply:
Done.